plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/util/util.py

```diff
@@ -27,10 +27,11 @@ def split_list_values(input_list: list, separator: str) -> list:
         output_list = []
         for item in input_list:
             if isinstance(item, str):
-                item_split = item.split(separator)
-                output_list.extend(item_split)
-        duplicates = set()
-        return [item for item in output_list if not (item in duplicates or duplicates.add(item))]
+                item_split = item.split(":")
+                if len(item_split) == 2:
+                    output_list.append({"hostname": item_split[0], "endpoint_id": item_split[1]})
```

Comment:
I'll update this in a new PR now, but the documentation doesnt provide any examples of what these can be, and Eric is unable to get any information from the customer since the ticket was opened a year ago 

I'm also not able to get data from the trigger myself